tests/integrationv2/providers.py

```diff
@@ -47,9 +47,9 @@ def __init__(self, options: ProviderOptions):
 
         self.options = options
         if self.options.mode == Provider.ServerMode:
-            self.cmd_line = self.setup_server()
+            self.cmd_line = self.setup_server() # lgtm [py/init-calls-subclass]
```

Comment:
It seems like inline suppression comments aren't supported in CodeQL: https://github.com/github/codeql/issues/9298

We should probably remove all of these and manually dismiss the alerts as "won't fix" in the UI.

Reply:
Is that just for go or everything in general? https://github.com/github/codeql/issues/3293 suggests that in-line alert suppression should work for python but it isn't applied on the first PR

Reply:
Ah got it okay. Let's try it then, and if they don't go away we can always dismiss them after merging.